Data Deletion Request Framework.md

```diff
@@ -311,13 +311,13 @@
       <tr>
       <td>identifierType</td>
       <td>string</td>
-      <td>The type of identifier to which the deletion request applies.The value in this field must be of a type as specified in the intended recipients' dsrdelete.json file.</td>
+      <td>The type of identifier provided in the identifierValue field. The value in this field must match a type specified in the intended recipients' dsrdelete.json file.</td>
       <td>required</td>
       </tr>
       <tr>
       <td>identifierFormat</td>
       <td>string</td>
-      <td>The format of identifier to which the deletion request applies.</td>
+      <td>The format of the identifier provided in the identifierValue field. The valur in this field must match a format specified in the intended recipients' dsrdelete.json file.</td>
```

Comment:
The second word in the second sentence has a typo: is "valur" should be "value".
The word "recipients'" in the second sentence has the apostrophe in the wrong place, should be before the S "recipient's" as there is only a single recipient. Suggest maybe rewriting the sentence as:
> The value in this field must match a format specified in the dsrdelete.json file of the intended recipient.